### ISSUE TITLE
this comment looks very strange

### ISSUE THREAD

**Issue by aric87** [THoj13/spotify-API-hack#2]
https://github.com/THoj13/spotify-API-hack/blob/master/index.html#L18

Looks like you commented out the carriage return so your code looked nicely formatted, but the button and input didn't have the 2 px space. I don't know if this is an issue, or an ingenious hack... 

I do think that an employer _might_ think its odd though. Consider changing, or, close the issue.

**Comment by THoj13** [comment]
Right, I remember this. I found it as an answer on Stack Overflow. Here is the [link](http://stackoverflow.com/questions/2095886/htmlcss-space-between-input-buttons) to it if you have any interest in it.
